### ISSUE TITLE
"from . import __path__" gives import error

### ISSUE THREAD

**Issue by BLLCHDHRY** [pasmopy/breast_cancer#43]
In step 11 (editing the Search Param Class), the line `from . import __path__` gives the error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: attempted relative import with no known parent package
```

I am using Python version 3.9.13 on Windows 11 to run the code. Would this possibly be causing issues? If so are there any solutions?

**Comment by himoto** [comment]
Did you edit the `set_search_param.py` file generated from Text2Model? Which code did you run to get this error message?

**Comment by BLLCHDHRY** [comment]
I have not intentionally attempted to edit set_search_param.py. I was following the protocol in https://doi.org/10.1016/j.xpro.2022.101619 and have gotten to the Construction of a comprehensive model of the ErbB signaling network section. I completed steps 9 and 10 without any obvious problems but could not complete step 11.
![image](https://github.com/user-attachments/assets/fad73e51-1f58-4709-89cd-b91d33b9f889)
For reference my current file directory looks like this:
![image](https://github.com/user-attachments/assets/fe1be170-12dc-4c01-a8ae-4d2b05c6a442)
![image](https://github.com/user-attachments/assets/37718df1-8739-4cca-b156-ebec289021b5)
![image](https://github.com/user-attachments/assets/5b53481d-687b-4ea2-9b22-2493e5a96b13)
![image](https://github.com/user-attachments/assets/d10dbe24-d42e-4715-9fa2-a8dc376af86c)

And also this:
![image](https://github.com/user-attachments/assets/38a2aac4-455b-496a-a23c-7a43237791dd)
![image](https://github.com/user-attachments/assets/4b12b213-a79b-4d7b-822a-cc1f5ced5f44)
![image](https://github.com/user-attachments/assets/e5bbadfd-efe7-4003-b85b-9c45e93c77fb)
![image](https://github.com/user-attachments/assets/d2a24d8f-238c-4738-bdcb-9a248ae3ca2e)

**Comment by himoto** [comment]
Please edit the `set_search_param.py` file as described in step 11. Running this code directly will return the error.

**Comment by BLLCHDHRY** [comment]
I have added the code to the `set_search_param.py` file instead of running it directly.
![image](https://github.com/user-attachments/assets/4c68e2da-0ae6-4e69-9779-d5c988dbc406)
![image](https://github.com/user-attachments/assets/c08dae4f-3acd-4f59-8f3f-ce413062af0d)
However I have run into a new problem in step 13. Attempting to run `mv erbb_network_jl training` fails and gives an error which I suspect is because there is already a erbb_network_jl file in the training folder. I have attempted to manually exchange them to move the new erbb_network_jl file from models to training but when running the optimisation it seems to fail.
